Sources/Plasma/FeatureLib/pfPython/cyPythonModule451.cpp

```diff
@@ -95,6 +80,44 @@ PYTHON_NO_INIT_DEFINITION(ptModulePackLoader)
 
 // ==========================================================================
 
+static pyObjectRef IInitModuleSpec(
+    const ST::string& modName,
+    const ST::string& packedName,
+    pyObjectRef loader,
+    bool isPackage
+)
+{
+    // Try to share the same python string object for the module name and
+    // the packed name, since they are often the same.
+    pyObjectRef modNameObj, packedNameObj;
+    if (modName.c_str() == packedName.c_str()) {
```

Comment:
Just confirming: Are you wanting to test for the strings being identical (same pointers) or the same (same values)?

Reply:
I'm testing for the same pointers. In the vast majority of cases, the same `ST::string` is passed in for the module and packed name.

```C++
IInitModuleSpec(name, name, self->fThis->fLoader, false);
```